### DISCUSSION TITLE
Builder in Nuxt 3

### DISCUSSION THREAD

**Discussion by hnviradiya** [nuxt/nuxt#16709] [Questions]
I am trying to serve nuxt app from nest js. We have `builder` in nuxt 2. But Unable to find a similar concept in Nuxt 3.

Is there a way to serve nuxt app from NestJs (or say expressjs app) using `builder` like following in nuxt 3?

The following repo serves both nuxt app and also API data from the same NestJs as the backend. Is it possible to do it in Nuxt 3?

https://github.com/Goopil/nest-nuxt-starter/blob/master/server/nuxt/NuxtServer.ts

**Comment by TheAlexLichter** [comment]
Likely related: https://github.com/nuxt/framework/discussions/1704

**Reply by hnviradiya** [reply]
Yes. But unsolved there as well.

**Reply by hnviradiya** [reply]
Do we have any solution to this?
